TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/LinearSlide.kt

```diff
@@ -0,0 +1,48 @@
+package org.firstinspires.ftc.teamcode.api
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode
+import com.qualcomm.robotcore.hardware.DcMotor
+
+/**
+ * An API for controlling the linearSlide.
+ */
+object LinearSlide : API() {
+    private lateinit var hook: DcMotor
+    private const val LINEAR_SLIDE_REDUCTION_SLOPE: Double = (0 - 0.8) / (4500 - 4250)
+
+    private object linearSlideConfig {
+        @JvmField
+        var min = 0.0
+
+        @JvmField
+        var max = 4500
+    }
```

Comment:
Can you put this in the `RobotConfig` file? Also please change the variable / object names to be consistent.

Reply:
fixed